models/event.py

```diff
@@ -226,7 +226,15 @@ def week(self):
             self._week = days / 7
 
         return self._week
-
+                          
```

Comment:
`Blank line contains whitespace`
https://travis-ci.org/the-blue-alliance/the-blue-alliance/jobs/494044321#L3238